CloudFoundry/content/apps/quotas.md

```diff
@@ -0,0 +1,114 @@
+---
+menu:
+  main:
+    parent: apps
+title: Quotas
+weight: -10
+---
+
+#### Quotas:
+
+Cloud Foundry capacity is managed by quota. Quotas provide a not to exceed reservation of memory, compute, application routes and service instances. 
+
+##### Quota allocations:
+
+We have allocated trial participants a default quota of 10GB of RAM, 1000 routes and 100 service instances.
+
+- 1000 application routes.
+- 10.024GB of RAM.
+- 100 service instances.
+
+###### Routes:
+
```

Comment:
The five `#` make all the titles (routes, ram etc.) look much smaller than the text of the paragraphs. I'd just use 4 hashes like with Quotas (line 9).